website/blog/2021-11-05-release.md

```diff
@@ -0,0 +1,135 @@
+---
+slug: release-0.19
+title: "New in Wave 0.19: Custom Colors"
+author: Martin Turoci
+author_title: Software Engineer @ H2O.ai
+author_url: https://github.com/mturoci
+author_image_url: https://avatars.githubusercontent.com/u/64769322?v=4
+tags: [release]
+---
+
+With the latest release, we are excited to announce Wave becoming more colorful! In addition to existing color themes (`default`, `neon`) we added `h2o-dark` and plan on expanding the list to at least a dozen
+so stay tuned. If interested, you can track the progress [here](https://github.com/h2oai/wave/issues/696). However, having only predefined color palettes is limiting so we decided to allow a custom, user-defined palettes.
+
+<!--truncate-->
+
+## How it works?
+
+Firstly, a new `ui.theme` needs to be registered within the `themes` attribute in `ui.meta_card`. It takes the name of the theme and 4 main colors as arguments.
+After the theme is successfully registered, you can use it in the same fashion as predefined themes, by setting `ui.meta_card`'s `theme` attribute.
+
+```py
+q.page['meta'] = ui.meta_card(
+    box='',
+    themes=[
+        ui.theme(
+            name='my-awesome-theme',
+            primary='#13ebe7',
+            text='#e8e1e1',
+            card='#12123b',
+            page='#070b1a',
+        )
+    ],
+    theme='my-awesome-theme'
+)
+```
+
+:::tip
+You can register multiple themes, which might be handy for switching between the light / dark mode of your app.
+:::
+
+## Theme generator
+
+Wave ships with a simple app that allows you to pick colors and see how they work together instantly. Simply run:
+
+```sh
+./venv/bin/wave run examples.theme_generator
+```
+
+Then visit <http://localhost:10101/demo>.
+
+![theme-generator](assets/2021-11-05/theme-generator.gif)
+
+This Wave app provides:
+
+* Immediate visual feedback.
+* Color contrast warnings when a particular contrast ratio is less than [4.5](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html).
+* Quick theme copy-pasting.
+
+## Can individual colors be changed?
+
+In general, the answer is no. Wave tries to be smart and do all the minor decisions for you, e.g. picking a color for primary/secondary buttons, checkboxes, etc. However, some cards allow changing colors of particular parts, for example, icons, but this is more of an exception rather than a strict rule.
+
+If full control over colors/typography etc. is needed, going with regular HTML/CSS/JS is probably a better choice.
+
+## Bunch of new cards added
```

Comment:
Suggestion to add a link to the example for each new card

Reply:
Agree, thanks!